system-test/translate.ts

```diff
@@ -15,41 +15,48 @@
  */
 
 import * as assert from 'assert';
-import {Translate} from '../src';
+import {TranslationServiceClient} from '../src';
 
 const API_KEY = process.env.TRANSLATE_API_KEY;
 
 describe('translate', () => {
-  let translate = new Translate();
+  const translate = new TranslationServiceClient();
 
   describe('detecting language from input', () => {
     const INPUT = [
       {
-        input: 'Hello!',
+        content: 'Hello!',
         expectedLanguage: 'en',
       },
       {
-        input: '¡Hola!',
+        content: '¡Hola!',
         expectedLanguage: 'es',
       },
     ];
 
     it('should detect a langauge', async () => {
-      const input = INPUT.map(x => x.input);
-      const [results] = await translate.detect(input);
-      assert.strictEqual(results[0].language, INPUT[0].expectedLanguage);
-      assert.strictEqual(results[1].language, INPUT[1].expectedLanguage);
+      const projectId = await translate.getProjectId();
+      for (const input of INPUT) {
+        const [result] = await translate.detectLanguage({
+          content: input.content,
+          parent: `projects/${projectId}`,
```

Comment:
It seems like a slightly more difficult to work with API surface, that we now need to provide a `parent`. I noticed this was also the case for `v3beta1` @nnegrey?

Reply:
Translate now has user managed resources and locations

Reply:
:+1:  so this is an expected change to the API? thought it might be a bug, and was talking to @alexander-fenster about it, sounds like it's not.

Reply:
The only difference with JS gapic-generator is that gapic-generator provides a helper function for this:

```js
const contents = [];
const targetLanguageCode = '';
const formattedParent = client.locationPath('[PROJECT]', '[LOCATION]'); 
//                             ^^^ we don't have this yet in TypeScript so
//                                 we need to concatenate a string manually
const request = {
  contents: contents,
  targetLanguageCode: targetLanguageCode,
  parent: formattedParent,
};
const [response] = await client.translateText(request);
```

We will provide the same interface soon (@xiaozhenliu-gg5 is working on it). But the fact that it now requires `project` to be passed is an API change, not much we can do here.